google/cloud/forseti/services/scanner/scanner.proto

```diff
@@ -24,6 +24,7 @@ service Scanner {
 }
 
 message RunRequest {
+  string handle = 1;
```

Comment:
Let's make this proto name more clear, how about `scanner_name`?

Reply:
modified as suggested